go.sh

```diff
@@ -6,9 +6,21 @@ export LOG_UUID=FALSE
 
 . /defaults.sh
 
+
 cat > ${NGIX_CONF_DIR}/server_certs.conf <<-EOF_CERT_CONF
     ssl_certificate     ${SERVER_CERT};
     ssl_certificate_key ${SERVER_KEY};
+    # Can add SSLv3 for IE 6 but this opens up to poodle
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
Be good to limit this to TLSv1.2 and we can PR for the others if specifically required.